oneoffs/oneoff_utils.py

```diff
@@ -1,75 +1,105 @@
-import numpy as np
 import os
-import pandas as pd
-import sgf
-import utils
-import utils
 
+import sgf
+from tensorflow import gfile
 from tqdm import tqdm
-from sgf_wrapper import sgf_prop, replay_sgf
+
+import dual_net
+import shipname
+from gtp_wrapper import MCTSPlayer
+from sgf_wrapper import replay_sgf, sgf_prop_get
+from utils import parse_game_result, logged_timer
 
 
 def get_sgf_props(sgf_path):
-  with open(sgf_path) as f:
-    sgf_contents = f.read()
-  collection = sgf.parse(sgf_contents)
-  game = collection.children[0]
-  props = game.root.properties
-  return props
+    # TODO(sethtroisi): move this to sgf_wrapper.
+    with open(sgf_path) as f:
+        sgf_contents = f.read()
+    collection = sgf.parse(sgf_contents)
+    game = collection.children[0]
+    props = game.root.properties
+    return props
 
 
 def parse_sgf(sgf_path):
```

Comment:
needs a docstring to distinguish why you'd use this instead of the stuff in sgf_wrapper (should this go in sgf_wrapper?)

Reply:
This can be completely replaced by sgf_wrapper. It was used by training_curve and blindly copied by me. I'll fix this after I submit this PR (which only changed the indention of this block)